docs(upgrade): updates upgrade of Cluster Operator watching multiple namespaces (#10319)
strimzi/strimzi-kafka-operator: documentation/modules/upgrading/proc-upgrade-cluster-operator.adoc

* An existing Cluster Operator deployment is available.
* You have xref:downloads-{context}[downloaded the release artifacts for Strimzi {ProductVersion}].
* You need an account with permission to create and manage `CustomResourceDefinition` and RBAC (`ClusterRole`, and `RoleBinding`) resources.

.Procedure

> . Take note of any configuration changes made to the existing Cluster Operator resources in the `/install/cluster-operator` directory.

Reviewer: This assumes you still have the directory which I doubt will be the case. Maybe we could rephrase it to be more generic?

```suggestion
. Take note of any configuration changes made during the previous Cluster Operator installation.
```
